src/middleware/authentication.ts

```diff
@@ -0,0 +1,22 @@
+import { AuthenticationError } from "apollo-server";
+import { logger } from "../logger";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+export const authenticationContextMiddleware = ({ req }: { req: any }) => {
```

Comment:
We should strive to avoid using any. You can probably infer what arguments are being passed to `authenticationContextMiddleware` by hovering over this line in `  context: authenticationContextMiddleware,` in app.tsx

Reply:
That is my bad, I knew that the type was Request, but forgot to add it. Now it is added.